aws/terraform/modules/metaflow/modules/metadata-service/ec2.tf

```diff
@@ -0,0 +1,111 @@
+resource "aws_security_group" "metadata_service_security_group" {
+  name        = local.metadata_service_security_group_name
+  description = "Security Group for Fargate which runs the Metadata Service."
+  vpc_id      = var.metaflow_vpc_id
+
+  ingress {
+    from_port   = 8080
+    to_port     = 8080
+    protocol    = "tcp"
+    cidr_blocks = [var.vpc_cidr_block]
+    description = "Allow API calls internally"
+  }
+
+  ingress {
+    from_port   = 8082
+    to_port     = 8082
+    protocol    = "tcp"
+    cidr_blocks = [var.vpc_cidr_block]
+    description = "Allow API calls internally"
+  }
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = -1
+    self        = true
+    description = "Internal communication"
+  }
+
+  # egress to anywhere
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1" # all
+    cidr_blocks = ["0.0.0.0/0"]
+    description = "Allow all external communication"
+  }
+
+  tags = merge(
+    var.standard_tags,
+    {
+      Metaflow = "true"
+    }
+  )
+}
+
+resource "aws_lb" "this" {
+  name               = "${var.resource_prefix}nlb${var.resource_suffix}"
+  internal           = true
+  load_balancer_type = "network"
+  subnets            = [var.subnet1_id, var.subnet2_id]
+
+  tags = var.standard_tags
+}
+
+resource "aws_lb_target_group" "this" {
```

Comment:
Refer to the issue below: https://github.com/Netflix/metaflow-tools/pull/26/files#r677874253. Hit this problem with character limit again with this resource when I tested on Gov Cloud.  Gov Cloud's region identifier is longer (`us-gov-west-1`), so public sector users will hit this limit both for this resource as well as the DB Target Group I called out below.